### DISCUSSION TITLE
[Platform.sh configurator] Update database DSN platform format

### DISCUSSION THREAD

**Discussion by apfelbox** [symfony/symfony#51740] [Q&A] [answered]
Hi,

the support from Platform.sh sent me here, to report this issue.

The format of the database DSN in the configurator needs to be updated:

<img width="1757" alt="Screenshot_2023-09-19_15 26 09" src="https://github.com/symfony/symfony/assets/1032411/df9feabb-bf93-4894-b167-601df8ea5f35">

**Comment by fabpot** [comment]
This should be fixed by https://github.com/symfony-cli/symfony-cli/pull/347
Closing here to keep the discussion on the PR itself.

**Comment by fabpot** [comment, marked as answer]
A new CLI version has now been released.

**Reply by apfelbox** [reply]
Thank you @fabpot 🙌
